build.sbt

```diff
@@ -14,8 +14,8 @@ val assemblySettings = Seq(
 )
 
 inThisBuild(Seq(
-  scalaVersion := "2.13.12",
-  crossScalaVersions := Seq("2.12.18", scalaVersion.value),
+  scalaVersion := "3.3.1",
+  crossScalaVersions := Seq("2.12.18", "2.13.12", scalaVersion.value),
```

Comment:
Non-blocking, but I wonder if we could look to drop `2.12` support soon?

Reply:
Based on https://metrics.gutools.co.uk/d/InMu69VIz/dependency-versions?orgId=1 it looks like we don't have any projects on Scala 2.12 that use Anghammarad, lets drop it now.